cmd/neofs-cli/modules/util/acl.go

```diff
@@ -276,7 +277,23 @@ func parseEACLRecord(args []string) (*eacl.Record, error) {
 			}
 
 			eacl.AddFormedTarget(r, role, pubs...)
+		case "account": // targets
+			var (
+				err      error
+				accounts []user.ID
+			)
 
+			if len(ss) == 2 {
```

Comment:
what if len is not 2? the whole `case` is senseless then (in contrast, the previous may miss the second arg)

Reply:
ss can be length 1 (if just `deny getrange address` without address and `:`). It is an invalid record. Changed the condition for error, in this case.

Otherwise ss have length 2, because `strings.SplitN(args[i], ":", 2)`
Even we have a record like `address:NWcSSzMkw5Vuq3gFmcih5yfCNxfXjWdKz8:NWcSSzMkw5Vuq3gFmcih5yfCNxfXjWdKz8` with two `:`. The result length is still 2:
* address
* NWcSSzMkw5Vuq3gFmcih5yfCNxfXjWdKz8:NWcSSzMkw5Vuq3gFmcih5yfCNxfXjWdKz8

This is just invalid format

Reply:
But it needs to be handled.